sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/QueueDescription.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 using Azure.Core;
-using Azure.Messaging.ServiceBus.Primitives;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
```

Comment:
Is this needed?

Reply:
Nope, it was this suggestion - 

![image](https://user-images.githubusercontent.com/45216704/83826315-7401d500-a690-11ea-97f5-34cf4684d087.png)

Reply:
No 😞